src/main/java/org/jitsi/impl/protocol/xmpp/XmppProtocolProvider.java

```diff
@@ -252,6 +252,8 @@ public boolean verify(String s, SSLSession sslSession)
         }
 
         connection = new XMPPTCPConnection(connConfig.build());
+        // Jibri can take a while to reply, so give a decent-sized timeout
+        connection.setReplyTimeout(60000);
```

Comment:
This affects the whole connection, so I'm kind of worried if it would break other things...

Reply:
Can we add a timeout argument to sendIqWithResponseCallback instead ?

Reply:
Ah! Didn't notice it had an overload with a timeout argument there.  Yeah I'll try that.